chains/ethereum/events.go

```diff
@@ -46,11 +46,58 @@ func (l *Listener) handleTransferEvent(eventI interface{}) msg.Message {
 	return msg
 }
 
-func (l *Listener) handleTestDeposit(eventI interface{}) msg.Message {
+func (l *Listener) handleErcTransferEvent(eventI interface{}) msg.Message {
+	log15.Debug("Handling deposit proposal event")
+	event := eventI.(ethtypes.Log)
+
+	contractAbi, err := abi.JSON(strings.NewReader(emitter.EmitterABI))
+	if err != nil {
+		log15.Error("Unable to decode event", err)
+	}
+
+	var ercEvent emitter.EmitterERCTransfer
+	err = contractAbi.Unpack(&ercEvent, "ERCTransfer", event.Data)
+	if err != nil {
+		log15.Error("Unable to unpack ERCTransfer", err)
+	}
+
+	// Capture indexed values
+	ercEvent.DestChain = event.Topics[1].Big()
+	ercEvent.DepositId = event.Topics[2].Big()
+
+	msg := msg.Message{
+		Type:        msg.CreateDepositProposalType,
+		Source:      l.cfg.id,
+		Destination: msg.ChainId(uint8(ercEvent.DestChain.Uint64())),
+		// TODO: Can we safely downsize?
+		DepositId: uint32(ercEvent.DepositId.Uint64()),
+		To:        ercEvent.To.Bytes(),
+		// Metadata:  ercEvent.Data,
```

Comment:
Why no meta?

Reply:
need to wait for #149 to see how this will be handled